### PR TITLE
ocr2keeper integration test

### PR DESCRIPTION
**Pull request by infiloop2**
Add integration test for ocr2keeper plugin

### CODE REVIEW

File: core/services/ocr2/plugins/ocr2keeper/integration_test.go [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+	"github.com/smartcontractkit/chainlink/core/services/ocr2/validate"
+	"github.com/smartcontractkit/chainlink/core/services/ocrbootstrap"
+	"github.com/smartcontractkit/chainlink/core/services/relay/evm"
+	"github.com/smartcontractkit/libocr/commontypes"
+	"github.com/smartcontractkit/libocr/networking"
+	"github.com/smartcontractkit/libocr/offchainreporting2/confighelper"
+	ocrTypes "github.com/smartcontractkit/libocr/offchainreporting2/types"
+	"github.com/stretchr/testify/require"
+	"github.com/umbracle/ethgo/abi"
+	"gopkg.in/guregu/null.v4"
```
Comment (begmaroman) [inline core/services/ocr2/plugins/ocr2keeper/integration_test.go:45]: Minor: imports should be sorted properly

Comment (infiloop2) [reply core/services/ocr2/plugins/ocr2keeper/integration_test.go:45]: i think they already are?

File: core/services/ocr2/plugins/ocr2keeper/integration_test.go [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+	// Add the bootstrap job
+	bootstrapNode.AddBootstrapJob(t, fmt.Sprintf(`
+		type               	= "bootstrap"
+		relay 				= "evm"
+		schemaVersion      	= 1
+		name               	= "boot"
+		contractID    	    = "%s"
+		contractConfigTrackerPollInterval = "1s"
+
+		[relayConfig]
```
Comment (begmaroman) [inline core/services/ocr2/plugins/ocr2keeper/integration_test.go:254]: Would be nice to allign rows

Comment (infiloop2) [reply core/services/ocr2/plugins/ocr2keeper/integration_test.go]: done

**Pull request merged**